pages/track-progress.html

```diff
@@ -0,0 +1,293 @@
+<html>
+
+
+<style>
+.dropbtn {
+    background-color: #61c2f9;
+    color: white;
+    padding: 16px;
+    font-size: 16px;
+    border: none;
+    cursor: pointer;
+}
+
+.dropdown {
+    position: relative;
+    display: inline-block;
+}
+
+.dropdown-content {
+    display: none;
+    position: absolute;
+    background-color: #f9f9f9;
+    min-width: 160px;
+    box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
+    z-index: 1;
+}
+
+.dropdown-content a {
+    color: black;
+    padding: 12px 16px;
+    text-decoration: none;
+    display: block;
+}
+
+.dropdown-content a:hover {background-color: #f1f1f1}
+
+.dropdown:hover .dropdown-content {
+    display: block;
+}
+
+.dropdown:hover .dropbtn {
+    background-color: #61c2f9;
+}
+</style>
+
+
+  <script>
+  
+  		
+		// Create a new request object
+var request = new XMLHttpRequest(); // global var
+
+
+
+
+//parseInt(request.responseText.substr(18,3))
+
+  
+  
+  //-------- Website
+    function goToURL() {
+		var user = prompt("Please enter username", "Dogi");
+
+  
+location.href = 'https://github.com/pulls?utf8=%E2%9C%93&q=is%3Apr+author%3A'+user+'+is%3Apublic+repo%3Aopen-learning-exchange%2Fopen-learning-exchange.github.io+is%3Amerged+'
+    }
+	
+	function goToURL2() {
+		var user = prompt("Please enter username", "Dogi");
+
+
+ 
+
+location.href ='https://github.com/pulls?utf8=%E2%9C%93&q=is%3Apr+author%3A'+user+'+is%3Apublic+repo%3Aopen-learning-exchange%2Fopen-learning-exchange.github.io'
+    }
+	
+	
+	
+	
+	
+	
+	//------------------------API
+	
+	function goToURL3() {
+	var user = prompt("Please enter username", "Dogi");
+    
+ 
+		
+		location.href =  'https://api.github.com/search/issues?q=repo:open-learning-exchange/open-learning-exchange.github.io+author:'+user+'+is:merged&sort=created&order=asc'
+		
+		
+		
+
+    }
+	function goToURL4() {
+	var user = prompt("Please enter username", "Dogi");
+    
+     
+		location.href =  'https://api.github.com/search/issues?q=repo:open-learning-exchange/open-learning-exchange.github.io+author:'+user+'+type:pr&sort=created&order=asc'
+				
+
+
+    }
+	
+		function goToURL5() {
+	var user = prompt("Please enter username", "Dogi");
+    
+     
+		location.href =  'https://api.github.com/search/issues?q=repo:open-learning-exchange/open-learning-exchange.github.io+author:'+user+'+type:issue&sort=created&order=asc'
+				
+
+
+    }
+	
+	function Total_PRs(){ //Test-ALL
+	var user = prompt("Please enter username", "Dogi");
+     
+		
+		var object= 'github.com';
+		object =  'https://api.github.com/search/issues?q=repo:open-learning-exchange/open-learning-exchange.github.io+author:'+user+'+type:pr&sort=created&order=asc'
+		console.log(object)
+	
+
+	
+// Initialize a request
+request.open('get', object)
+request.send()
+		request.onreadystatechange = function() {
+    if (this.readyState == 4 && this.status == 200) {
+      alert("Number of PRs:"+request.responseText.substr(18,3))	
```

Comment:
@paulbert Thank you , now we don't have to worry about commas popping up in the alert.